docs/modeling/new-model.rst

```diff
@@ -277,3 +277,24 @@ input``.
 
     The above example is essentially equivalent to the built-in
     `~astropy.modeling.functional_models.Linear1D` model.
+
+
+Advanced Notes
+==============
+
+The Model class implements a custom metaclass, ``modeling.core._ModelMeta``,
+which handles a number of modifications to the standard class creation
+process. For instance, user-defined ``inverse`` properties are overwritten,
+changing it to ``_inverse`` so that ``Model.inverse`` is always used (it calls
+``_inverse``).
+
+Subclassing this MetaClass, allows for further customization. This is not encouraged since ``_ModelMeta`` is currently a private class and is free to be changed at whim.
```

Comment:
Please wrap this line.

Reply:
Actually, per [this comment](https://github.com/astropy/astropy/pull/11017#issuecomment-736006874) I don't think this should be documented at all, nor should there be a changelog entry.

If there's a case where you want to be able to prevent wrapping `Model.inverse` maybe that can be an option directly to `_ModelMeta`.  I don't think we should be making anything about `_ModelMeta` publicly documented unless we were to make it a public method in the first place and document explicit use cases for subclassing it.

Reply:
> Actually, per this comment I don't think this should be documented at all, nor should there be a changelog entry.

I agree. I've removed the documentation.

> f there's a case where you want to be able to prevent wrapping Model.inverse maybe that can be an option directly to _ModelMeta.

Maybe for a future PR. 🤞